slam-ws/src/nav_stack/launch/move_base.launch

```diff
@@ -4,6 +4,11 @@
 
   <master auto="start"/>
 
+  <!-- Remap cmd_vel topic name from keyboard teleop -->
+  <node name="teleop_twist_keyboard" pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py">
+    <remap from="cmd_vel" to="key_vel"/>
+  </node>
```

Comment:
Can we make a separate `launch` file for teleop keyboard? I don't think we need teleop to run every time we run `move_base` but rather we should call it if we need it